app/celeryapp/__init__.py

```diff
@@ -0,0 +1,32 @@
+from celery import Celery
+
+celery = None
+CELERY_TASK_LIST = [
+    "app.tasks",
+]
+
+
+def make_celery(app):
+    celery = Celery(
+        app.import_name,
+        # backend=app.config['CELERY_RESULT_BACKEND'],
+        # broker=app.config['CELERY_BROKER_URL'],
+        include=CELERY_TASK_LIST
+    )
+    celery.conf.update(app.config)
+    celery.conf.update(
+        broker_url="redis://localhost:6379/0",
+        result_backend="db+sqlite:///results.sqlite",
```

Comment:
why couldnt we place this in the base app `__init__.py` file unlike creating a new package?

Reply:
I was making some references to this repository https://github.com/kwiersma/flask-celery-sqlalchemy and adapt the code to fit into my application. I will try to place the Celery app configs in the base app and see how it works.

Reply:
> I was making some references to this repository https://github.com/kwiersma/flask-celery-sqlalchemy and adapt the code to fit into my application. I will try to place the Celery app configs in the base app and see how it works.

that is  fine and  okay
but let these references just be a source of inspiration and understanding and not a  way  to  enforce you to structure the project